config.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+import configparser
+import logging
+import os
+
+
+def parse_arguments(args):
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--log-level', metavar="LEVEL", type=str, help="Log level", default=None)
+    parser.add_argument('rest_args', metavar="ARGS", nargs='*')
+    ns = parser.parse_args(args)
+    return vars(ns)
+
+
+def parse_configfile():
+    cfg = configparser.ConfigParser()
+    cfg.read(os.path.expanduser("~/.rtorrent_low_space_driver.cf"))
+    return cfg
+
+
+def start_logger(ns, cfg):
+    log_level = ns.get('log_level') or cfg.get('main', 'log_level', fallback='INFO')
+    logging.basicConfig(
+        level=getattr(logging, log_level),
+        format="%(asctime)s - %(levelname)8s - %(name)s - %(message)s"
+    )
+
+
+class MyConfiguration:
```

Comment:
I'd rename this to just `Configuration` I think, or `ConfigurationImpl`, something along those lines.  `My` makes it sound too much like a demo.

Reply:
Dear Mr Banks, agreed, it should be named Configuration. 


As for this PR, I haven't asked your opinion on this? Should configparsing be on a separate module? Or do you feel it fits better being another class/functions in driver.py?

Reply:
> Do softwares that load from configurarions from files typically have a separate module for doing so?

Yeah, I'd say that's a pretty common setup.  This refactor's definitely an improvement.